projects/js-packages/publicize-components/src/components/connection-management/style.module.scss

```
@@ -33,7 +33,9 @@
 	cursor: default;
 }
 
-
+.confirmDialog > div > div > div > span {
```

Do you think this could be a little fragile, say if the markup changes?

I tried this and it works

```suggestion
.confirmDialog span {
```